pkg/apply/merge.go

```diff
@@ -0,0 +1,178 @@
+package apply
```

Comment:
ditto, let's investigate if we can re-use code from https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/apply/apply.go or similar.